Code/src/main/java/nl/utwente/group10/ui/components/lines/Connection.java

```diff
@@ -31,13 +37,13 @@ public Connection() {
     }
 
     public Connection(OutputAnchor from) {
-        this.setStartAnchor(from);
-        setEndPosition(from.getCenterInPane());
+        tryAddAnchor(from);
+        setEndPosition(getScenePoint(from));
```

Comment:
En nu zie ik dat de getScenePoint ook verandert was. Dat is het probleem inderdaad. Heel raar, want dit ging eerst wel goed.